packages/terra-clinical-data-grid/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Clinical DataGrid
+
+The DataGrid is an organizational component that renders a collection of data in a grid-like format.
```

Comment:
Nitpick:

Should we mention Clinical datagrid, or are we not too concerned about specifying it's namespace as it exists in this repo?

Reply:
The other components within terra-clinical are referred to without the 'Clinical' moniker within their docs/class names/etc. It seems like the package name is the main differentiator.